website/docs/docs/cloud-integrations/semantic-layer/gsheets.md

```diff
@@ -81,17 +77,12 @@ queryBuilder="/img/docs/dbt-cloud/semantic-layer/gsheets-query-builder.jpg"
 
 ## Using saved queries
 
-<p>Access <a href="/docs/build/saved-queries">saved queries</a>, powered by MetricFlow, in {props.type} to quickly get results from pre-defined sets of data. To access the saved queries in {props.type}:</p>
-
-<ol>
-  <li>Open the hamburger menu in {props.type}.</li>
-  <li>Navigate to <strong>Saved Queries</strong> to access the ones available to you.</li>
-  <li>You can also select <strong>Build Selection</strong>, which allows you to explore the existing query. This won't change the original query defined in the code.
-    <ul>
-      <li>If you use a <code>WHERE</code> filter in a saved query, {props.type} displays the advanced syntax for this filter.</li>
-    </ul>
-  </li>
-</ol>
+Access <a href="/docs/build/saved-queries">saved queries</a>, powered by MetricFlow, to quickly get results from pre-defined sets of data. To access the saved queries in your integration:
+
+- Open the hamburger menu in Google Sheets.
+- Navigate to **Saved Queries** to access the ones available to you.
+- You can also select **Build Selection**, which allows you to explore the existing query. This won't change the original query defined in the code.
+  - If you use a `where` filter in a saved query, Google Sheets displays the advanced syntax for this filter.
```

Comment:
Hiya @mirnawong1 

This is a non-blocker suggestion but would it be a good idea to link to the where page: https://docs.getdbt.com/reference/resource-configs/where?

Kind Regards
Natalie

Reply:
the where here is a filter, not a config. however i can add a link to here: /docs/build/saved-queries#where-clause